### PR TITLE
Updated readme to include project and branch description

### PR DESCRIPTION
**Pull request by padobariya**
Updated readme to include project and branch description

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,23 @@
...
+# Overview
 
 This is a simple example Apollo Android app inspired by https://github.com/apollographql/frontpage-ios-app.
+It is a very basic app, but it does demonstrate how you can hook up GraphQL query results to your UI using MVVM architecture.
+
+The code in `PostsRepository.kt` fetches data based on a GraphQL query defined in `PostListActivity.graphql`. That query refers to a fragment defined in `PostListItem.graphql`, which nicely illustrates how you can describe your data needs next to the UI component that uses them.
+
+## Workflow
+
+We followed following workflow while working GraphQL endpoint and the Apollo SDK.
```
Comment (aruke) [inline README.md]:
```suggestion
We followed the following workflow while working GraphQL endpoint and the Apollo SDK.

```

File: README.md [hunk 2]
```diff
@@ -1,6 +1,23 @@
...
+
+- Write and test queries using [GraphiQL](https://github.com/graphql/graphiql).
+- Use the Apollo plugin to generate model types from the schema and queries.
+- Integrate ApolloClient into the app, passing in the generated queries and write the logic for handling already-parsed responses.
+
+## Branch Info
+
+Branch | Description
+:--: | :--:
+master | The master branch contains the GraphQL implementation using RxJava2 support  |
```
Comment (aruke) [inline README.md]: Basically every branch is going to have GraphQL implementation so better to skip that part.
Just add `RxJava2 Support`.
For other branches, we can add a short description like above.

**Review by aruke — CHANGES_REQUESTED** [review]
I added some comments.

**Comment by padobariya** [comment]
@aruke Thanks! Addressed 👍

**Review by aruke — APPROVED** [review]
Looks good now!

**Review by oh-naoki — APPROVED** [review]
⭐ LGTM ⭐

**Pull request merged**